siliconcompiler/tools/klayout/klayout_export.py

```diff
@@ -268,3 +269,6 @@ def read_fills(top):
 
 gds_export(design, in_def, in_gds, out_gds, tech_file, foundry_lef, macro_lefs,
            config_file='', seal_file='', timestamps=sc_timestamps)
+
+# Generate screenshot
+show(schema, out_gds, f'outputs/{design}.png', screenshot=True)
```

Comment:
might be nice to wrap this with a generate_screenshot var (default to true), but allow us to control